go/libkb/home.go

```diff
@@ -43,6 +45,7 @@ type HomeFinder interface {
 	ServiceSpawnDir() (string, error)
 	SandboxCacheDir() string // For macOS
 	InfoDir() string
+	IsNonstandardHome() (bool, error)
 }
 
 func (b Base) getHome() string {
```

Comment:
Unrelated to this PR but having both `Home` and `getHome` in this struct hierarchy is weird.

Did I get it correctly, that `getHome` is solely for getting "configured home" via either config or command line?

Reply:
Sounds about right